test/functional/test-article.spec.ts

```diff
@@ -13,33 +25,46 @@ describe('Get /article/id', () => {
   });
 
   test('Can get an article as XML', async () => {
+    await populateCollection('articles', [{...article}]);
     return agent
       .get('/articles/54296')
       .set('Accept', 'application/xml')
       .expect('Content-Type', /xml/)
-      .expect(200);
+      .expect(200)
+      .then(response => {
+        expect(response.text).toBe('<article></article>');
+    });
   });
 
   test('Can get an article as JSON', async () => {
+    await populateCollection('articles', [{...article}]);
     return agent
       .get('/articles/54296')
       .set('Accept', 'application/json')
       .expect('Content-Type', /json/)
-      .expect(200);
+      .expect(200)
+      .then(response => {
+        expect(response.body).toEqual(expect.objectContaining(article));
+    });
   });
 
   test('By default article is returned as JSON', async () => {
+    await populateCollection('articles', [{...article}]);
     return agent
       .get('/articles/54296')
       .expect('Content-Type', /json/)
       .expect(200);
   });
 
   test('Can get article manifest', async () => {
+    await populateCollection('articles', [{...article, xml: '<article><body><fig id="fig1" position="float"><graphic xlink:href="elife-54296-fig1.tif" mimetype="image" mime-subtype="tiff"/></fig></body></article>'}]);
     return agent
       .get('/articles/54296/manifest')
       .expect('Content-Type', /json/)
-      .expect(200);
+      .expect(200)
+      .then(response => {
+        expect(response.body).toEqual({"articleId": "54296", "assets": [{"id": "fig1", "path": "http://localhost:8080/api/v1/articles/54296/assets/elife-54296-fig1.tif", "type": "image/jpeg"}], "path": "/api/v1/articles/54296/export", "type": ""});
```

Comment:
I notice here that the resource is `.tif` but the mime type is `image/jpeg`. Is that expected?

Reply:
This seems to be hard coded to `"type": "image/jpeg"` in the article service `getManifest`. I'll raise an issue to make type detectable from the file.